### DISCUSSION TITLE
ESP32-Faikin integration with automation solution other than HomeAssistant

### DISCUSSION THREAD

**Discussion by cmonty14** [revk/xESP32-Faikout#573] [General]
Hi,
can you confirm that ESP32-Faikin integration with automation solution other than HomeAssistant is working?

My automation solution in place is Loxone that is enhanced with LoxBerry, a plugin platform for Raspberry Pi.

THX

**Comment by revk** [comment]
It supports a number of MQTT based messages and a web interface, and the legacy Daikin web interface messages.

There is so special "integration" for other applications, but any that use the legacy Daikin web interface may work.=

**Comment by cmonty14** [comment]
Well, my assumption is that legacy Daikin WebUI is not available anymore with newer Wifi modules. Can you confirm?
I can control my AC only with Daikin app "ONECTA", but I cannot ping IP from my local network assigned to AC.

**Reply by revk** [reply]
New Daikin WiFI modules do not have the legacy Daikin http interface, but many integrations in existing systems have it as it used to work and works with older controllers, so most of the legacy Daikin http interface is implemented in the Faikin to allow for this.

I have no idea on ONECTA or how the new Daikin WiFi modules work, sorry.
